data/modules/CargoRun/CargoRun.lua

```diff
@@ -0,0 +1,1138 @@
+})
+spaceship_parts = Equipment.EquipType.New({
+	l10n_key = 'SPACESHIP_PARTS', slots="cargo", price=250,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+titanium = Equipment.EquipType.New({
+	l10n_key = 'TITANIUM', slots="cargo", price=150,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+tungsten = Equipment.EquipType.New({
+	l10n_key = 'TUNGSTEN', slots="cargo", price=125,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+uranium = Equipment.EquipType.New({
+	l10n_key = 'URANIUM', slots="cargo", price=175,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+quibbles = Equipment.EquipType.New({
+	l10n_key = 'QUIBBLES', slots="cargo", price=1,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+wedding_dresses = Equipment.EquipType.New({
+	l10n_key = 'WEDDING_DRESSES', slots="cargo", price=15,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+
+local chemical = {}
+table.insert(chemical, digesters)
+table.insert(chemical, hazardous_substances)
+
+local mining = {}
+table.insert(mining, clus)
+table.insert(mining, explosives)
+
+local hardware = {}
+table.insert(hardware, aluminium_tubes)
+table.insert(hardware, diamonds)
+table.insert(hardware, hazardous_substances)
+table.insert(hardware, machine_tools)
+table.insert(hardware, neptunium)
+table.insert(hardware, plutonium)
+table.insert(hardware, semi_finished_products)
+table.insert(hardware, spaceship_parts)
+table.insert(hardware, titanium)
+table.insert(hardware, tungsten)
+table.insert(hardware, uranium)
+
+local infrastructure = {}
+table.insert(infrastructure, clus)
+table.insert(infrastructure, explosives)
+table.insert(infrastructure, greenhouses)
+
+local consumer_goods = {}
+table.insert(consumer_goods, electrical_appliances)
+table.insert(consumer_goods, furniture)
+table.insert(consumer_goods, spaceship_parts)
+
+local expensive = {} -- price >= 175
+table.insert(expensive, art_objects)
+table.insert(expensive, diamonds)
+table.insert(expensive, neptunium)
+table.insert(expensive, plutonium)
+table.insert(expensive, spaceship_parts)
+table.insert(expensive, uranium)
+
+local fluffy = {}
+table.insert(fluffy, quibbles)
+
+local wedding = {}
+table.insert(wedding, wedding_dresses)
+
+local art = {}
+table.insert(art, art_objects)
+
+local gems = {}
+table.insert(gems, diamonds)
+
+local radioactive = {}
+table.insert(radioactive, neptunium)
+table.insert(radioactive, plutonium)
+table.insert(radioactive, uranium)
+
+local centrifuges = {}
+table.insert(centrifuges, aluminium_tubes)
+table.insert(centrifuges, machine_tools)
+table.insert(centrifuges, titanium)
+
+local custom_cargo = {
+	{ bkey = "CHEMICAL", goods = chemical, weight },
```

Comment:
Still. In this case, ", weight }" means "Add the value of the symbol weight to the array part of the table", where weight is nil, so basically it is a NOP. I'd suggest "weight = 1" or whatever the default value should be.